### PR TITLE
updated dockerfile

### PR DESCRIPTION
**Pull request by naveen-armedia**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+
+LABEL ORG="Arkcase LLC" \
       VERSION="1.0" \
       IMAGE_SOURCE=https://github.com/ArkCase/ark_cloudconfig \
       MAINTAINER="Armedia LLC"
+#################
+# Build JDK
+#################
+
+ARG JAVA_VERSION="11.0.12.0.7-0.el7_9"
```
Comment (bojmil-armedia) [inline Dockerfile]: Config Server uses Java 8. Does it work with Java 11 as well?

Comment (naveen-armedia) [reply Dockerfile]: yeah, updated the version to 1.8.0.

File: Dockerfile [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US:en \
+    LC_ALL=en_US.UTF-8
+
+RUN yum update -y && \
+    yum -y install java-1.8.0-openjdk-devel-${JAVA_VERSION} unzip && \
+    $JAVA_HOME/bin/javac -version
+#################
+# Build ConfigServer
+#################
```
Comment (david-oc-miller) [inline Dockerfile:23]: The config server needs some configuration (like: the URL to the message broker (ActiveMQ or Kafka), and it needs the configuration files.  Does the orchestration layer (Helm chart or docker-compose file) take care of these things?

Comment (naveen-armedia) [reply Dockerfile:23]: Yes, @david-oc-miller   all that details will be defined in the application.yaml and we can take care of that either in the docker-compose file or Helm chart.

**Comment by naveen-armedia** [comment]
> Hi Naveen... it should be a capital C, e.g. "ArkCase LLC".

yeah noted @david-oc-miller

**Review by david-oc-miller — APPROVED** [review]

**Pull request merged**